doc/man1/pkeyutl.pod

```diff
@@ -55,6 +57,19 @@ Print out a usage message.
 This specifies the input filename to read data from or standard input
 if this option is not specified.
 
+=item B<-rawin>
+
+This indicates that the input data is raw data, which is not hashed by any
+message digest algorithm. The user can specify a digest algorithm by using
+the B<-digest> option.
```

Comment:
We should say that this option can only be used with -sign and -verify

Reply:
Okay, added